File: 167-two-sum-ii-input-array-is-sorted/167-two-sum-ii-input-array-is-sorted.ts
const twoSum = (numbers: number[], target: number) : number[] => {
  let left:number = 0;
  let right:number = numbers.length - 1;
  while (left < right) {
    const current:number = numbers[left] + numbers[right];
    if (current === target) return [left + 1, right + 1];
    if (current < target) left += 1;
    if (current > target) right -= 1;
  }
  return [-1, -1]
};

File: 167-two-sum-ii-input-array-is-sorted/167-two-sum-ii-input-array-is-sorted.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        d = {}
        for i, n in enumerate(numbers):
            diff = target - n
            if diff in d:
                return [d[diff], i + 1]
            d[n] = i + 1
                
                
            